nemo/utils/exp_logging.py

```diff
@@ -123,12 +123,12 @@ def __init__(
         # Create work_dir if specified
         if work_dir:
             self.work_dir = work_dir
-            self.ckpt_dir = os.path.join(self.work_dir, 'checkpoints')
             # only create tm_sur dir if checkpoints dir is not present in the work_dir
-            if add_time and not os.path.exists(self.ckpt_dir):
```

Comment:
Could you please pull from the latest master? or this will revert the latest changes

Reply:
Yesterday I checked the whole process.It was not a good idea to check for checkpoints folder. It was not matched the whole process designed in nemo. We ar uploading checkpoints on NGC. We just put the files, not the whole folder. So when user download from NGC, they can not load it with that mechanism unless they create fake folders for the main and checkpoints folder. So that process was not working actually.

I read other examples in other collections. Whenever we want to load and continue training, we should explicitly pass the checkpoint_dir. So the clean way is to add ckeckpoint_dir to the input arguments with default of empty. When somebody wants to load already trained models, they should pass the folder to the checkpoints. I updated the joint_intent_slot toturial. Take a look at that.